esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.html

```diff
@@ -114,18 +115,24 @@
               {$ node.power_state + (node.target_power_state ? " → " + node.target_power_state : "") $}
             </div>
           </td>
-          <td class="rsp-p2">{$ node.maintenance $}</td>
+          <td class="rsp-p2" ng-style="{'color': node.maintenance === 'True' ? 'red' : 'green'}">
```

Comment:
I'd suggest creating a `getMaintenanceColor` function and using that here; I know that the code is simple, but it creates a consistency (and makes it easier to update the colors in one place if needed later).

Reply:
It was very simple so I did this way for other two I did with functions only will do it that for maintenance too.